dali/kernels/signal/resampling_gpu.cuh

```diff
@@ -88,7 +88,7 @@ __global__ void ResampleGPUKernel(const SampleDesc *samples) {
     // position, keeping the floats small in order to keep precision. `in_block_f`, used to
     // calculate the reference for distance (in_block_i) needs to be calculated in double precision.
     double in_block_f = out_block_start * scale;
-    int64_t in_block_i = floor(in_block_f);
+    int64_t in_block_i = ::floor(in_block_f);
```

Comment:
How did this end up being necessary in this PR?

Reply:
because when you include vec.h, there's a `dali::floor` defined. It complains that there is no overload for float, because it tries to use that `dali::floor`